Editor/Util.cs

```diff
@@ -29,11 +29,17 @@
 using UnityEditor;
 using UnityEditor.Animations;
 using UnityEngine;
+
+#if MA_VRCSDK3_AVATARS
 using VRC.SDKBase.Editor.BuildPipeline;
+#endif
+
 using Object = UnityEngine.Object;
 
 namespace nadena.dev.modular_avatar.core.editor
 {
+
+#if MA_VRCSDK3_AVATARS
     internal class CleanupTempAssets : IVRCSDKPostprocessAvatarCallback
```

Comment:
This is dead code and can be deleted.

Reply:
That would be another task before this one.

Reply:
#515